### PR TITLE
Added Radium

### PR DESCRIPTION
**Pull request by AnthonyBloomer**

### CODE REVIEW

File: Casks/radium.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'radium' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://catpigstudios.com/Radium/updates/Radium.zip'
+  name 'Radium'
+  homepage 'http://catpigstudios.com'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  app ''
```
Comment (vitorgalvao) [inline Casks/radium.rb]: Missing app.

**Comment by Ngrd** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

File: Casks/radium.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'radium' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://catpigstudios.com/Radium/updates/Radium.zip'
+  name 'Radium'
+  homepage 'http://catpigstudios.com'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/radium.rb]: `:commercial`.

**Comment by vitorgalvao** [comment]
Ah, I was under the impression I had seen this cask before. [I did](https://github.com/caskroom/homebrew-cask/pull/6219). [We can’t accept it](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#trial-versions).

**Pull request closed without merging**